src/platform/apollolake/apollolake.x.in

```diff
@@ -486,7 +486,7 @@ SECTIONS
     . = . + HEAP_BUFFER_SIZE;
     _buffer_heap_end = ABSOLUTE(.);
 
-    . = ALIGN (SRAM_BANK_SIZE);
+    . = ALIGN (32);
```

Comment:
We should probably use the CACHE_LINE_SIZE macro here ?

Reply:
Previously (before SRAM_BANK_SIZE change) it was 32 everywhere, thats why I have reverted it on apl to 32.
CACHE_LINE_SIZE is 64? Why it is better?

Reply:
It provides better memory/cache performance since it aligns with memory IO burst size and cache line size.

Reply:
@lgirdwood I will give it a try then

Reply:
@lgirdwood works very well and also page size alignment for first buffer is not needed now, thanks